webserver/preferences.js

```diff
@@ -619,3 +619,85 @@ document.addEventListener("DOMContentLoaded", () => {
         });
     }
 });
+
+/**
+ * Sanity Data form before Save 
+ */
+function sanityData() {
+    var txt_error = "";
+    let errorMessage = document.getElementById('error-message');
+    const displayB = document.getElementById("DisplayBright");
```

Comment:
Please avoid using cryptic names for variables such as "displayB". Variable names should be self-explanatory.